src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+const SEX_MALE = 'm';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+
+      {isMarried ? (
+        <p className="Person__partner">
+          {`${partnerName} is my ${sex === SEX_MALE ? 'wife' : 'husband'}`}
```

Comment:
Let's create a variable for wife/husband to avoid nesting ternary operators
![image](https://github.com/mate-academy/react_person/assets/96409563/b9f69fe8-3643-400b-8a1a-4c1c9ba1edfb)